### DISCUSSION TITLE
Is there a way to support Initiating User Registration via OpenID Connect 1.0

### DISCUSSION THREAD

**Discussion by ThisIsMissEm** [goauthentik/authentik#10632] [Q&A]
This has a request that looks like:
```
 GET /as/authorization.oauth2?response_type=code
     &client_id=s6BhdRkqt3
     &state=tNwzQ87pC6llebpmac_IDeeq-mCR2wLDYljHUZUAWuI
     &redirect_uri=https%3A%2F%2Fclient%2Eexample%2Eorg%2Fcb
     &scope=openid%20profile
     &prompt=create HTTP/1.1
  Host: authorization-server.example.com
```

The key part is the `prompt=create`, the idea is that this would display the `default-enrollment-flow` before returning the user through the OAuth Flow. The use case is for signalling that the client application wants to create a user account with the Authorization Server

**Comment by asalimonov** [comment]
I agree with this request, as it should simplify the use case where Authentik is used for registering new users.
I tested a scenario involving the interception of a redirection to the identification flow and replacing it with an enrollment flow. It works as expected. I hope there  just need to pass a parameter and configure the behavior in Authentik in a way that was described by @ThisIsMissEm .
